src/metabase/driver/util.clj

```diff
@@ -198,6 +198,13 @@
       :placeholder (trs "You can use patterns like auth* to match multiple {0}" (str/lower-case disp-name))
       :visible-if {(keyword type-prop-nm) ["inclusion" "exclusion"]}}]))
 
+(defn find-schema-filters-prop
+  "Finds the first property of type `:schema-filters` for the given `driver` connection properties. Returns `nil`
+  if the driver has no property of that type."
+  [driver]
+  (first (filter (fn [conn-prop]
+                   (= :schema-filters (keyword (:type conn-prop))))
+           (driver/connection-properties driver))))
```

Comment:
I'm looking at `driver/connection-properties` and i see implementations only for h2, mysql, and postgres. Presumably redshift goes through the postgres version because of the derivation but I'm surprised there's no default behavior or more implementations.

Reply:
ah, nevermind i see the docstring about the lazy loading driver.